### DISCUSSION TITLE
training

### DISCUSSION THREAD

**Discussion by legolasek** [Woolverine94/biniou#16] [Ideas]
A tab for training voice models would be useful. I can't even locate the folder containing these models to be able to add the ready-made .npz for bark

**Comment by Woolverine94** [comment]
Hello @legolasek,

First of all, thanks for your interest in biniou. 

Voice cloning is indeed not supported for bark in biniou. It's a very basic implementation of bark that only use models provided by bark and bark-small. 

These models (in .npy format) can be found in your user space cache folder (e.g. : ~/.cache/huggingface/hub/models--ylacombe--bark-small/snapshots/<some_UID>/speaker_embeddings/v2 for example). I suppose you can replace it with yours, but I haven't tested it. 

Voice cloning is a feature considered for integration in biniou, not necessarily using bark (CoquiTTS seems more straightforward and easy to use for voice cloning, but I still have to test it yet).

However, Bark voice cloning in biniou is not completely discarded, but is not a priority either.
